go.mod

```diff
@@ -6,4 +6,5 @@ require (
 	github.com/aws/aws-sdk-go v1.32.3
```

Comment:
We should ensure the rest of these dependencies are also up to date as the aws sdk and mux are both behind.

Reply:
All dependencies updated